modules/components/src/AdvancedFacetView/SearchBox.js

```diff
@@ -224,60 +225,17 @@ export default class extends React.Component {
               },
               () => {
                 const newFilteredList = getCombinedFilteredList();
-                this.setState({
```

Comment:
Do we need these callbacks? could we refactor it to not need them?

Reply:
yes, this will be cleaned up after as part of reimplementation of the searchbox (a separate issue :) )